src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -48,7 +68,8 @@ public static IResourceBuilder<PostgresServerResource> AddPostgres(this IDistrib
                       {
                           context.EnvironmentVariables[UserEnvVarName] = postgresServer.UserNameReference;
                           context.EnvironmentVariables[PasswordEnvVarName] = postgresServer.PasswordParameter;
-                      });
+                      })
+                      .WithAnnotation(new HealthCheckAnnotation($"{postgresServer.Name}_check"));
```

Comment:
Naming of healthchecks coudl be better here. It might actually be better to have a WithHealthcheck method (perhaps internal initially) which takes a callback to configure the health check.

Reply:
Decided we can't really do better here (or rather there is no benefit). At the end of the day you still need to compute the name of the health check, so this may as well be it.